packages/commonwealth/server/models/community.ts

```diff
@@ -157,6 +158,7 @@ export default (
         allowNull: true,
         defaultValue: null,
       },
+      namespace: { type: dataTypes.STRING, allowNull: true },
```

Comment:
We'll need a migration for this.

Reply:
I had written the migration a while back when we were doing v1. So namespace already exists on "Communities"